app/src/main/java/net/gsantner/markor/format/general/CommonTextActions.java

```diff
@@ -55,9 +61,16 @@
     private final Activity _activity;
     private final HighlightingEditor _hlEditor;
 
+    protected AppSettings _appSettings;
+    private int _tabWidth;
+    protected Context _context;
+
     public CommonTextActions(Activity activity, HighlightingEditor hlEditor) {
         _activity = activity;
         _hlEditor = hlEditor;
+        _context = activity != null ? activity : _hlEditor.getContext();
+        _appSettings = new AppSettings(_context);
+        _tabWidth = _appSettings.getTabWidth();
```

Comment:
I am deriving _tabWidth from settings, while the existing indent function uses `_hlEditor.getShiftWidth`. `getShiftWidth` appears to search for strings such as `"sw=2"` in the text and will default to 4 if these are not found. I am assuming using the app settings value is the correct thing to do here.

Reply:
ok